pkg/etcdutil/etcdutil.go

```diff
@@ -259,13 +259,10 @@ func TakeAndSaveFullSnapshot(ctx context.Context, client client.MaintenanceClose
 	logger.Infof("Total time taken by Snapshot API: %f seconds.", timeTaken.Seconds())
 
 	if cc.Enabled {
-		startTimeCompression := time.Now()
 		rc, err = compressor.CompressSnapshot(rc, cc.CompressionPolicy)
 		if err != nil {
 			return nil, fmt.Errorf("unable to obtain reader for compressed file: %v", err)
 		}
-		timeTakenCompression := time.Since(startTimeCompression)
-		logger.Infof("Total time taken in full snapshot compression: %f seconds.", timeTakenCompression.Seconds())
```

Comment:
Is it possible to add this to the go routine that does the actual compression so that we still have this log?

Reply:
hmm ... let me test it.

Reply:
With inline compression, I found it is impossible to separate the time taken to read the snapshot reader from the pure compression time.